src/components/FindMovie/FindMovie.tsx

```diff
@@ -13,45 +77,64 @@ export const FindMovie: React.FC = () => {
           <div className="control">
             <input
               data-cy="titleField"
+              name="input"
               type="text"
               id="movie-title"
               placeholder="Enter a title to search"
               className="input is-dander"
+              value={searchTitle}
+              onChange={(event) => {
+                setSearchTitle(event.target.value);
+              }}
             />
           </div>
 
-          <p className="help is-danger" data-cy="errorMessage">
-            Can&apos;t find a movie with such a title
-          </p>
+          {isError && (
+            <p className="help is-danger" data-cy="errorMessage">
+              Can&apos;t find a movie with such a title
+            </p>
+          )}
         </div>
 
         <div className="field is-grouped">
           <div className="control">
             <button
               data-cy="searchButton"
               type="submit"
-              className="button is-light"
+              disabled={!searchTitle.trim().length || false}
+              className={
+                classNames(
+                  'button is-light',
+                  { 'is-loading': isLoading },
+                )
+              }
+              onClick={sendQuery}
             >
               Find a movie
             </button>
           </div>
 
-          <div className="control">
-            <button
-              data-cy="addButton"
-              type="button"
-              className="button is-primary"
-            >
-              Add to the list
-            </button>
-          </div>
+          { movie && (
+            <div className="control">
+              <button
+                data-cy="addButton"
+                type="button"
+                className="button is-primary"
+                onClick={() => addToList()}
+              >
+                Add to the list
+              </button>
+            </div>
+          )}
         </div>
       </form>
 
-      <div className="container" data-cy="previewContainer">
-        <h2 className="title">Preview</h2>
-        {/* <MovieCard movie={movie} /> */}
-      </div>
+      { movie && (
+        <div className="container" data-cy="previewContainer">
+          <h2 className="title">Preview</h2>
+          { movie && <MovieCard movie={movie} />}
```

Comment:
```suggestion
          <h2 className="title">Preview</h2>
          
          <MovieCard movie={movie} />
```
You've already checked if movie is present above